### PR TITLE
chore: Update issue templates

### PR DESCRIPTION
**Pull request by JanCizmar**

### CODE REVIEW

**Review by stepan662 — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+** Versions and environmen **
```
Comment (marketachalupnikova) [inline .github/ISSUE_TEMPLATE/bug_report.md:26]:
```suggestion
** Versions and environment **
```

File: .github/ISSUE_TEMPLATE/roadmap_item.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    - [ ] e2e
+  - [ ] ... Specify the test cases here, so you don't forget about important stuff ...
+- [ ] I've checked that all endpoints are well secured
+- [ ] The swagger documentation is OK including descriptions
+- [ ] The documentation is updated
+- [ ] Telemetry data is sent to PostHog
+
+## Before release checklist
+- [ ] Blog post about this new feature
+- [ ] Announcement in tolgee APP
```
Comment (marketachalupnikova) [inline .github/ISSUE_TEMPLATE/roadmap_item.md:39]:
```suggestion
- [ ] Announcement in Tolgee App
```

File: .github/ISSUE_TEMPLATE/roadmap_item.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+## After release checklist
+- [ ] Roadmap is updated
+- [ ] We posted about this in social media
+  - [ ] Twitter 
+  - [ ] Linkedin
+  - [ ] Facebook
+  - [ ] Reddit
+  - [ ] Slack Community
+  - [ ] Github Discussions
+- [ ] Pricing is updated
```
Comment (marketachalupnikova) [inline .github/ISSUE_TEMPLATE/roadmap_item.md:51]: add also Pricing in App is updated

Comment (JanCizmar) [reply .github/ISSUE_TEMPLATE/roadmap_item.md:51]: I think this is natural

**Review by ZuzanaOdstrcilova — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/roadmap_item.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    - [ ] Backend
+    - [ ] e2e
+  - [ ] ... Specify the test cases here, so you don't forget about important stuff ...
+- [ ] I've checked that all endpoints are well secured
+- [ ] The swagger documentation is OK including descriptions
+- [ ] The documentation is updated
+- [ ] Telemetry data is sent to PostHog
+
+## Before release checklist
+- [ ] Blog post about this new feature
```
Comment (JanCizmar) [inline .github/ISSUE_TEMPLATE/roadmap_item.md:38]:
```suggestion
- [ ] Preview is created & user stories are tested on the preview 
- [ ] Blog post about this new feature
```

**Pull request merged**